examples/knative/go/kn-go-harbor-webhook/README.md

```diff
@@ -0,0 +1,219 @@
+# kn-go-harbor-webhook
+
+Example Knative Go function for receiving [Project Harbor
+webhook](https://goharbor.io/docs/1.10/working-with-projects/project-configuration/configure-webhooks/)
```

Comment:
Link is referencing to an old (`v1.10`) documentation page. Event naming has changed for Harbor version `v2.5x`. Please use https://goharbor.io/docs/2.5.0/working-with-projects/project-configuration/configure-webhooks/

Reply:
Good catch! Using "latest" now to always ref latest docs.